.changeset/three-falcons-grow.md

```diff
@@ -0,0 +1,9 @@
+---
+"docs-app-for-embroider-css-modules-temporary": minor
+"test-app-for-embroider-css-modules": minor
+"docs-app-for-embroider-css-modules": minor
+"test-app-for-sample-v2-addon": minor
+"sample-v2-addon": minor
```

Comment:
I indicated a minor release as I want to do a major release alongside `embroider-css-modules`.